prow/cluster/statusreconciler_deployment.yaml

```diff
@@ -30,7 +30,7 @@ spec:
       terminationGracePeriodSeconds: 180
       containers:
       - name: statusreconciler
-        image: gcr.io/k8s-prow/statusreconciler:v20181129-31c544b
+        image: gcr.io/k8s-prow/status-reconciler:v20181220-899fabe
```

Comment:
Please use the same tag as the rest of prow: https://github.com/kubernetes/test-infra/pull/10673